api/CharacterData.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "after": {
```

Comment:
The data for `after` and `before` seems right, mdn-bcd-collector doesn't suggest any changes.